frontend/src/components/pages/admin/DisplayAssessmentsPage.tsx

```diff
@@ -142,6 +145,33 @@ const DisplayAssessmentsPage = (): React.ReactElement => {
   const [search, setSearch] = React.useState("");
   const [sortProperty, setSortProperty] = React.useState("name");
   const [sortOrder, setSortOrder] = React.useState("ascending");
+  const [types, setTypes] = React.useState<Array<string>>([]);
+  const [countries, setCountries] = React.useState<Array<string>>([]);
+  const [regions, setRegions] = React.useState<Array<string>>([]);
+  const [status, setStatus] = React.useState("");
+
+  const typeOptions = [
```

Comment:
can you move this into `src/types/AssessmentTypes.ts`? also maybe make an enum for BEGINNING and END in that file and update the values to use that enum (and also refactor the field for AssessmentTypes.type)

Reply:
Done!